apps/api/src/.env.production

```diff
@@ -7,6 +7,7 @@ DISABLE_USER_REGISTRATION=false
 WIDGET_BASE_URL=https://widget.novu.co
 NEST_STARTER_MAIL=support@novu.co
 CLIENT_SUCCESS_AUTH_REDIRECT=https://web.novu.co/auth/login
+STORE_ENCRYPTION_KEY="<ENCRYPTION_KEY_MUST_BE_32_LONG>"
```

Comment:
@djabarovgeorge could you please remove it from here? and from .env.development ? Want to make sure we raise the appropriate error if it's not exist, so that by mistake this generic value will be used.